capabilities_matrix/_topics/smartstate_analysis.md

```diff
@@ -24,6 +24,7 @@
 | XFS         | Linux     | ✅            |
 | ReiserFS    | Linux     | ✅            |
 | CDfs        | Linux     | ✅            |
+| RPM (RHEL 6, 7, 8, 9) | Linux | ✅      |
```

Comment:
@qasim-ahmed Can you move this down to the File Formats section, not the File Systems section?  RPM is already down below